<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogclerk</class>
 <widget class="QDialog" name="dialogclerk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>-50</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>员工号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>销户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎您，</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>761</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/osuser.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushfindnumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>点击查看手机号码</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_name_2</zorder>
  <zorder>label_2</zorder>
  <zorder>pushfindnumber</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
